devicon.json

```
@@ -2569,7 +2599,28 @@
         "color": "#F7931E",
         "aliases": []
     },
-    {
+	{
+        "name": "msdos",
+        "tags": [
+            "os",
```

Please remove the `,`
```suggestion
            "os"
```